app/css/interceptor.css

```diff
@@ -1165,3 +1165,45 @@ summary:where(details[open] > *) {
 	text-overflow: ellipsis;
 	overflow: hidden;
 }
+
+.columnar {
+	--grid-cols: minmax(min-content,max-content) minmax(50%,1fr);
+	--gap-x: 1rem;
+	--gap-y: 0.5rem;
+	font-size: 0.875rem;
+	line-height: 1.2;
+	width: fit-content;
+	margin: 0 auto;
+}
+
+.columnar > * {
+	max-width: 100%;
+	min-width: 0;
+}
+
+.columnar dt {
```

Comment:
can we come up with some better name for this?

Reply:
how about `key-value-pair`?